hurumap_land/sql/redistributedlandinhectares.sql

```diff
@@ -60,4 +81,3 @@ ALTER TABLE ONLY public.redistributedlandinhectares
 --
```

Comment:
`redistributedlandinhectares` should be `redistributedland` in primary key (`pk_redistributedlandinhectares`)